TF_Settings_Web/src/App.tsx

```diff
@@ -16,27 +16,27 @@ const App: React.FC = () => {
     const [tfStatus, setTfStatus] = React.useState<TrackingServiceState>(TrackingServiceState.UNAVAILABLE);
 
     useEffect(() => {
-        const updateTfStatus = () => {
-            ConnectionManager.RequestServiceStatus((detail: ServiceStatus) => {
-                const status = detail.trackingServiceState;
-                if (status) {
-                    setTfStatus(status);
-                }
-            });
-        };
-
         ConnectionManager.init();
 
-        ConnectionManager.AddConnectionListener(updateTfStatus);
-        const controller: WebInputController = new WebInputController();
+        const requestServiceStatus = () => ConnectionManager.RequestServiceStatus((detail: ServiceStatus) => {
+            const status = detail.trackingServiceState;
+            if (status) {
+                setTfStatus(status);
+            }
+        });
 
-        const timerID = window.setInterval(updateTfStatus, 5000);
+        const updateTfStatus = (serviceStatus:TrackingServiceState) => {
```

Comment:
We don't need the updateTfStatus method any more, I think you can just pass in the setTfStatus where it is used instead